resources/schemas/dbscripts/postgresql/targetedms-21.000-21.001.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE targetedms.transitionpredictionsettings ADD CONSTRAINT FK_TransitionPredictionSettings_PredictorCe FOREIGN KEY (CePredictorId) REFERENCES targetedms.Predictor(Id);
```

Comment:
To be extra cautious about potentially orphaned values in existing DBs which would cause upgrades to fail, let's clear out any bogus references that may be present. Before adding the FKs, do something like:

UPDATE targetedms.transitionpredictionsettings SET CePredictorId = NULL WHERE CePredictorId NOT IN (SELECT Id FROM targetedms.Predictor);